shared/settings/invites/index.desktop.tsx

```diff
@@ -24,6 +24,11 @@ class Invites extends Component<Props, State> {
       showMessageField: props.showMessageField,
     }
   }
+
+  componentDidMount() {
+    this.props.onRefresh()
```

Comment:
why add this?

Reply:
the container used to do this using recompose: https://github.com/keybase/client/pull/20953/files#diff-c14348ae521b4dd883e4c51536e266d8L47